test/lib.bash

```diff
@@ -270,6 +270,22 @@ function gather_knative_state {
     --dest-dir "$gather_dir" > "${gather_dir}/gather-knative.log"
 }
 
+function check_serverless_alerts {
+  logger.info 'Checking Serverless alerts'
+  local alerts_file monitoring_route num_alerts
+  alerts_file="${ARTIFACT_DIR:-/tmp}/alerts.json"
+  monitoring_route=$(oc -n openshift-monitoring get routes alertmanager-main -oyaml -ojsonpath='{.spec.host}')
```

Comment:
Will this work in CRC too? If you don't know, I can help doublecheck.

Reply:
Yeah. I don't know. ...if you could check that would be great.

Reply:
The route doesn't exist in a CRC cluster (In fact: It seems like the entire alert management system doesn't exist). Can we make room for that here? I.e. we can key off of `$OPENSHIFT_CI` being set or key off of the route not being there. I lean towards the former as that wouldn't "hide" issues if the route is renamed or whatever. Thoughts?

Reply:
Thanks. OK. Something like this?
```
[ -n "$OPENSHIFT_CI" ] && check_serverless_alerts
```

Reply:
yep